app/services/edge_cache/service.rb

```diff
@@ -0,0 +1,115 @@
+module EdgeCache
+  class Service
+    PROVIDERS = %w[fastly nginx].freeze
```

Comment:
Set this up so that we will have some guards in place should we add more caching providers in the future!